src/awx_plugins/credentials/injectors.py

```diff
@@ -9,14 +9,22 @@
 from awx_plugins.interfaces._temporary_private_container_api import (  # noqa: WPS436
     get_incontainer_path,
 )
+from awx_plugins.interfaces._temporary_private_credential_api import (  # noqa: WPS436
+    Credential,
+    GenericOptionalPrimitiveType,
+)
 from awx_plugins.interfaces._temporary_private_django_api import (  # noqa: WPS436
     get_vmware_certificate_validation_setting,
 )
 
 import yaml
 
 
-def aws(cred, env, private_data_dir):
+def aws(
+    cred: Credential,
+    env: dict[str, GenericOptionalPrimitiveType],
+    private_data_dir: str,
+) -> None:
```

Comment:
FWIW, I recommend reverting the hack in #64 and adding docstrings to all of these instead.